prober/utils.go

```diff
@@ -119,3 +126,14 @@ func ipHash(ip net.IP) float64 {
 	h.Write(ip)
 	return float64(h.Sum32())
 }
+
+func internationalizeDNSTarget(logger log.Logger, target string) (string, error) {
+	idnaTarget, err := idna.ToASCII(target)
```

Comment:
Should we use the lookup profile here rather than the raw punycode?

Reply:
Updated to use idna.Lookup.ToASCII